tests/common/include/tests/common/material_interaction.inl

```diff
@@ -276,11 +276,11 @@ TEST(material_interaction, telescope_geometry_energy_loss) {
     const scalar mass = interactor_state.mass;
 
     // new momentum
-    const scalar newP = state._stepping._bound_params.charge() /
-                        state._stepping._bound_params.qop();
+    scalar newP = state._stepping._bound_params.charge() /
+                  state._stepping._bound_params.qop();
 
     // new energy
-    const scalar newE = std::sqrt(newP * newP + mass * mass);
+    scalar newE = std::sqrt(newP * newP + mass * mass);
```

Comment:
There is btw also ```std::hypot```. The performance should not be much of an issue in testing code and as far as I know, it guards against some floating point pitfalls, too

Reply:
I changed it to hypot